bitrise-io/getting-started-react-native-sample-app
# Build speed optimizations (#6)

* Upgrade to react-native 0.71.1

* Update bitrise.yml

* Update jest to latest

* Remove Flipper

* Add react-native-inhibit-warnings

* Compiler optimizations to build faster

* Revert "Compiler optimizations to build faster"

This reverts commit c24160844fbb2d41cc7b736efe611b83c4084a88.

* Update bitrise.yml

Co-authored-by: Olivér Falvai <ofalvai@gmail.com>

* Suppress warning about pod master repo

* Add title to test step

* Remove step which is no longer needed

* Update to recommended way of disabling Flipper for CI builds

https://fbflipper.com/docs/getting-started/troubleshooting/react-native/#disable-flipper-on-ci-builds-ios

* Suppress warnings in Pods project

* Save cache after build

* Try code optimization again

* Remove redundant trigger

* Enable grade caching

* Build for Android release variant

* Update ruby to 3.2.0

* Update bitrise.yml to use the fastest machines

* Build for debug for both iOS and Android

---------

Co-authored-by: Olivér Falvai <ofalvai@gmail.com>
